src/CqlBuilderPanel/codesSection/codesSubSection/savedCodesSubSection/SavedCodesSubSection.tsx

```diff
@@ -400,32 +428,6 @@ export default function SavedCodesSubSection({
             </div>
           )}
         </tbody>
-
-        <Popover
-          optionsOpen={optionsOpen}
-          anchorEl={anchorEl}
```

Comment:
`anchorEl` ref variable declared at the top might be unused. if yes, I think can remove that as well.